src/boot/axios.js

```diff
@@ -9,17 +9,15 @@ import axios from 'axios';
 // for each client)
 const api = axios.create({ baseURL: '/api' });
 
-/**
- * Redirect on Leto-modelizer on status 503, otherwise throw error.
- * @param {Error} error - Error from axios.
- */
-function manageError(error) {
-  if (error.response.status !== 503) {
-    throw error.response.data;
-  }
-  window.location.href = process.env.LETO_MODELIZER_URL;
-}
-
+api.interceptors.response.use(
+  (response) => Promise.resolve(response),
```

Comment:
Missing:
`({ data }) => Promise.resolve(data),`

Reply:
Cannot do that, I need `response.data` and `response.headers` for `getPictureByLogin` method